# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/mine/dialog-1.0.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mine-0.20/selector.c	2004-11-04 10:53:31.000000000 +0100
+++ mine-0.20-p/selector.c	2004-11-08 04:32:45.206956448 +0100
@@ -21,6 +21,7 @@
 #include <dialog.h>
 #include <limits.h>
 #include <sys/ioctl.h>
+#include <stdio.h>
 
 int screensize_x = 70;
 int screensize_y = 19;
@@ -535,12 +536,12 @@
 		screensize_y = ws.ws_row - 3;
 	}
 
-	init_dialog();
+	init_dialog(stdin, stdout);
 
 	/* Yup - that's a label for goto.... */
 main_loop:
 	while (1) {
-		put_backtitle();
+		dlg_put_backtitle();
 
 		for (p=packages; p != NULL; p = p->next) {
 			p->is_dependency=0;
